starter-code/src/clue.js

```diff
@@ -92,3 +105,31 @@ var roomsArray = [];
 
 // Weapons Collection
 var weaponsArray = [];
+
+
+var characters = {}
+
+charactersArray.push(mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard)
+
+roomsArray.push(rooms)
```

Comment:
This is an unnecessary step, you already have an array so there's no need to push it into an empty array.